README.md

```diff
@@ -108,6 +110,42 @@ type RouteClient struct {
 func GetRoute(destination string) *Route { ... }
 ```
 
+## The case of an initialism should be consistent
+
+In code, the case of an initialism should be consistent, i.e. either all
+uppercase or lowercase. For example, "TCP" should be written as "TCP" or "tcp",
+e.g. `ProtocolTCP`, `tcpHeader`.
+
+In comments and documentation, initialisms should be all uppercase, e.g.
+"Protocol could be UDP, TCP, or SCTP".
+
+```go
+// Bad example
+// TcpHeader contains the tcp specific header.
+type TcpHeader struct { ... }
+
+func (p *Packet) SetTcpHeader(Tcp *TcpHeader) { ... }
+
+// Good example
+// TCPHeader contains the TCP specific header.
+type TCPHeader struct { ... }
+
+func (p *Packet) SetTCPHeader(tcp *TCPHeader) { ... }
+```
+
+## Use upper camel case for K8s resource and CRD kind names
+
+In comments and documentation, use upper camel case for K8s resource and CRD
+kind names, e.g. "Pod", "Service", "TrafficControl".
```

Comment:
Could we add Node and NetworkPolicy too. They are used a lot in Antrea, and people often use different formats.

Reply:
added